demos/list.html

```diff
@@ -1329,6 +1329,48 @@ <h3>Example - Interactive List (Dark Theme)</h3>
             evt.preventDefault();
           });
         }
+
+        var deactivateRippleHandler = null;
+        var checkboxListRippleSurfaceClass = 'checkbox-list-ripple-surface';
+
+        function clickHandler(evt, rippleFoundation, li) {
+          var checkbox = li.querySelector('.mdc-checkbox__native-control');
+          var isLabel = Array.prototype.some.call(checkbox.labels, function(label) { return label === evt.target; });
```

Comment:
`checkbox.labels` is not supported in IE 11 💩  or Edge 🙁

Reply:
:( I'll use querySelector instead

Reply:
reference for myself: https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/labels